Source/JavaScriptCore/parser/SourceProvider.h

```diff
@@ -206,23 +206,25 @@ class UnlinkedFunctionCodeBlock;
     };
 
     // RAII class for managing a Wasm source provider's underlying buffer.
```

Comment:
Can you fix this comment?  It's no longer just Wasm source.

Reply:
Fixed!